packages/graphql/tests/performance/performance.json

```diff
@@ -187,10 +215,10 @@
             "maxRows": 35,
             "dbHits": 214,
             "cache": {
-                "hits": 25,
+                "hits": 31,
                 "misses": 0
             },
-            "time": 19
+            "time": 6
```

Comment:
question: We don't store the cypher test query results right? Is that something we should consider?

Reply:
not for now, it can easily be added if we agree we need it